packages/react/src/core/builder.utils.ts

```diff
@@ -504,6 +543,9 @@ export function getCollectionDefaultFindManyApiRoute(args: {
         defaultApi: defaultHandler as any,
         page: payload.query.page ?? 1,
         pageSize: payload.query.pageSize ?? 10,
+        search: payload.query.search,
+        sortBy: payload.query.sortBy === '__id' ? 'id' : payload.query.sortBy,
```

Comment:
There's no guarantee that `__id` is `id` of the table. There's an options that can change `__id` reference. It's called `identifierColumn` which located in `fields.config.identifierColumn`

Also we have `__pk` which determine primary key of the table (Not support composite pk yet.). 

So, please change to this.





```suggestion
        sortBy: payload.query.sortBy === '__id' ? fields.config.identifierColumn : 
          payload.query.sortBy === '__id' ? fields.config.primaryColumn // not sure
          :  payload.query.sortBy,
```